soup-ranking.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"os"
+	"net/http"
+	"log"
+	"github.com/gorilla/mux"
+	"github.com/sjuls/soup-ranking/routes"
+	"github.com/sjuls/soup-ranking/dbctx"
+)
+
+func main() {
+	port := os.Getenv("PORT")
+	database := os.Getenv("DATABASE_URL")
+
+	if err := dbctx.Init(&database); err != nil {
```

Comment:
Sometimes people hide the panic functionality in functions like `InitOrDie` with no return value